vms/platformvm/client.go

```diff
@@ -326,6 +330,69 @@ func (c *client) GetCurrentValidators(
 	return getClientPermissionlessValidators(res.Validators)
 }
 
+// SubnetOnlyValidator is the response from calling GetSubnetOnlyValidator on
+// the API client.
+type SubnetOnlyValidator struct {
+	SubnetID              ids.ID
+	NodeID                ids.NodeID
+	PublicKey             *bls.PublicKey
+	RemainingBalanceOwner *secp256k1fx.OutputOwners
+	DeactivationOwner     *secp256k1fx.OutputOwners
+	StartTime             uint64
+	Weight                uint64
+	MinNonce              uint64
+	// Balance is the remaining amount of AVAX this SoV has to pay the
```

Comment:
(nit) "has for paying"

It could just be my complete lack of context but "has to pay" led my brain down the path of "is required to pay [entity]".